modules/cli/cmds/test.go

```diff
@@ -19,7 +19,7 @@ type Test struct {
 	BundleInputArgs
 	Quiet       bool              `usage:"Just print the match and don't print the resources" short:"q"`
 	Group       string            `usage:"Cluster group to match against" short:"g"`
-	ClusterName string            `usage:"Cluster name to match against" short:"n"`
+	ClusterName string            `usage:"Cluster name to match against"`
```

Comment:
I suspect ClusterName will be a popular option. What would you think about using capital 'N' as a short?

Reply:
I am not sure if Capital flag is good idea here. @nickgerace Thoughts?

Reply:
We could use `-c`, but I agree that capital flags are uncommon enough that we should only use them when necessary.

Reply:
It looks like we already have `-L`. To make it consistent I will rename to it `name` and shortflag it as `-N`